test/Twig/PheatureFlagsExtensionTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Test\Community\Symfony\DependencyInjection;
+
+use Pheature\Community\Symfony\Twig\PheatureFlagsExtension;
+use Pheature\Community\Symfony\Twig\PheatureFlagsRuntime;
+use Pheature\Core\Toggle\Read\ConsumerIdentity;
+use Pheature\Core\Toggle\Read\Feature;
+use Pheature\Core\Toggle\Read\FeatureFinder;
+use PHPUnit\Framework\TestCase;
+use Pheature\Core\Toggle\Read\Toggle;
+use Pheature\Core\Toggle\Read\ToggleStrategies;
+use Pheature\Core\Toggle\Read\ToggleStrategy;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+use Twig\RuntimeLoader\FactoryRuntimeLoader;
+
+final class PheatureFlagsExtensionTest extends TestCase
+{
+    public function testItShouldExposeTwoTests(): void
+    {
+        $extension = new PheatureFlagsExtension(new Toggle($this->createStub(FeatureFinder::class)));
+        self::assertCount(2, $extension->getTests());
```

Comment:
```suggestion
        $this->assertCount(2, $extension->getTests());
```
We decide some time ago to use always $this to maintain the same syntax in all PHPUnit API usage https://github.com/pheature-flags/toggle-core/pull/49

Reply:
Fixed